File: remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i = 0, j = 0, sz = nums.size();
        if (!sz)
            return 0;
        while (j < sz) {
            if(nums[i] == nums[j])
                ++j;
            else
                nums[++i] = nums[j++];
        }
        return 1+i;
    }
};

File: remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        slow, fast = 0, 0
        for fast in range(len(nums)):
            if nums[slow] != nums[fast]:
                nums[slow+1] = nums[fast]
                slow = slow+1
        
        return slow+1